prefq/examples/imitation_preference_comparisons.py

```diff
@@ -221,14 +221,20 @@ def _wait_for_feedback_request(url):
     ),
     env=venv,
     n_steps=2048 // venv.num_envs,
+    clip_range=0.1,
```

Comment:
Please add a comment to the top of the file that indicates why we chose these hyperparameters (a link to the PR in which I propose them should be fine).